composeApp/src/commonMain/kotlin/com/arkivanov/minesweeper/game/GameState.kt

```diff
@@ -98,5 +98,6 @@ internal val CellStatus.isFlagged: Boolean
 internal fun Cell.open(): Cell =
     copy(status = CellStatus.Open)
 
+// TODO: Can we somehow increase work of this thing? not each time count run for all grid
```

Comment:
Let's change the TODO text to something like "Consider making this a part of the state"